src/style.css

```diff
@@ -1 +1,64 @@
 /* add styles here */
+.body{
+  margin: 0;
+}
+
+
+
+.stars {
+  display: flex;
+
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
+.stars__star{
+  background-image: url('./images/star.svg');
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+
+}
+
+.stars__star:last-child{
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n+1){
+  background-image: url('./images/star-active.svg');
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--2 .stars__star:nth-child(-n+2){
+  background-image: url('./images/star-active.svg');
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
```

Comment:
Don't repeat yourself. Combine the selectors and change only property that is changed